config/locales/en/providers.yml

```diff
@@ -268,6 +268,7 @@ en:
       show:
         banner_text: The merits information was copied from %{application_ref}.
         h1-heading: Check your answers
+        inset_text: You cannot change the answers on this page once you save and continue.
```

Comment:
Is it worth moving this to the `generic` block so it's only stored onece, then it can be used on pages with 
```ruby
t(".generic.inset_text")
```

Reply:
Thanks, I have just updated the text to be stored in `generic.yml` and have added a line to the CYA feature test (as suggested by @jsugarman )